lib/evm.ex

```diff
@@ -6,23 +6,28 @@ defmodule EVM do
   # σ[a]
   @type account :: %{
           # σ[a]n
-          :nonce => integer(),
+          nonce: integer(),
           # σ[a]b
-          :balance => integer(),
+          balance: integer(),
           # σ[a]s
-          :storage => MerklePatriciaTree.Trie.t(),
+          storage: MerklePatriciaTree.Trie.t(),
           # σ[a]c
-          :code => binary()
+          code: binary()
         }
   # σ
   @type world_state :: %{
-          binary() => account()
+          address() => account()
         }
   @type trie_root :: MerklePatriciaTree.Trie.root_hash()
   @type val :: integer()
   @type address :: <<_::160>>
   @type hash :: <<_::256>>
   @type timestamp :: integer()
+  @type log_entry :: %{
+    address: address(),
+    topics: [],
```

Comment:
this is a list of what?

Reply:
@hayesgm it's list of integers. I'll add type, thanks. 

this pr is not finished yet. I encountered bug in `calldatacopy` operation - https://github.com/exthereum/evm/issues/46. That's why tests are failing.

Would you mind reviewing it one more time when it's finished? I hope I can finish it today or by monday in worst case scenario.